pop.css

```diff
@@ -0,0 +1,151 @@
+.pop-up-background {
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px);
+}
+
+.pop-up-card {
+  border: 1px solid #dfe1e6;
+  border-radius: 4.26vw;
+  background-color: #fff;
+  width: 91.46vw;
+  overflow: auto;
+}
+
+.pop-up-card h3 {
+  font-size: 2rem;
+  margin-top: 4.8vw;
+  margin-bottom: 0;
+}
+
+.quit-button {
+  background-image: url(images/quit\ Icon.png);
+  position: relative;
+  left: 80.267vw;
+  top: 8vw;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 0.75rem;
+  height: 0.75rem;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.pop-up-card ul.company {
+  margin: 5.867vw 5.3vw;
+}
+
+.pop-up-card .categories p {
+  font-size: 0.75rem;
+}
+
+.pop-up-card .company li {
+  margin: 0 3.2vw 0 0;
+}
+
+.pop-up-card .company li p {
+  font-size: 0.813rem;
+}
+
+.pop-up-desk-layout .right {
+  display: flex;
+  flex-direction: column;
+}
+
+.pop-up-img {
+  align-self: center;
+  width: 90%;
+}
+
+.pop-up-card hr {
+  border: 1px solid #ebecf0;
+  align-self: center;
+  width: 90%;
+  margin: 0;
+}
+
+.button-container {
+  justify-content: space-around;
+}
+
+.pop-up-card .div-button {
+  margin-top: 3.038vh;
+}
+
+.pop-up-card .button-container button {
+  padding: 1.701vh;
+}
+
+.button-icon {
+  margin: 0 0.625rem;
+  height: 1.25rem;
+}
+
+@media (min-width: 768px) {
+  .pop-up-card {
+    border-radius: 1.1vw;
+    width: 80.278vw;
+  }
```

Comment:
- [ ] This may help you solve it, kindly play with height of this pop-up-card untill it fits the design. note that this is just a suggestion but you can fix it the way you was just make sure the cancel icon is present in large screens like mine and the space on top and bottom of the pop-up-card.


https://github.com/pulgosodog/Portfolio-template-1/assets/65068771/cfbd3b85-9b73-4df6-80d4-651b3efb9b8b

Reply:
Finished! The height was changed to a responsive value and the quit (x) button is available on more screen sizes.
🙌